examples/notify-turn-hook.py

```
@@ -0,0 +1,248 @@
+
+        # Find last assistant message
+        asst_entries = [e for e in entries if e.get("type") == "assistant"]
+        if not asst_entries:
+            return 0
+        last_asst = asst_entries[-1]
+
+        user_ts = parse_timestamp(last_user["timestamp"])
+        asst_ts = parse_timestamp(last_asst["timestamp"])
+
+        return int((asst_ts - user_ts).total_seconds())
+    except Exception as e:
```

Ruff (0.14.6)

Do not catch blind exception: `Exception`

(BLE001)